Assets/Scripts/Managers/HouseManager.cs

```diff
@@ -18,13 +18,18 @@ public enum HouseMode { Explore, Decorate }
 	private List<HouseItem> houseItems;
 	private float houseValue = 0;
 
+	private AudioManager audioManager;
+
```

Comment:
Can be removed - since the manager is a singleton, you don't need a reference to the script to call it

Reply:
resolved